test/unit/api/java/CMakeLists.txt

```diff
@@ -0,0 +1,54 @@
+#####################
+## CMakeLists.txt
+## Top contributors (to current version):
+##   Mudathir Mohamed
+## This file is part of the CVC5 project.
+## Copyright (c) 2009-2021 by the authors listed in the file AUTHORS
+## in the top-level source directory and their institutional affiliations.
+## All rights reserved.  See the file COPYING in the top-level source
+## directory for licensing information.
+##
+
+find_package(Java REQUIRED)
+include(UseJava)
+find_package(JUnit REQUIRED)
+
+# specify source files for junit tests
+set(java_test_src_files
+  ${CMAKE_CURRENT_SOURCE_DIR}/cvc5/SolverTest.java
+)
+
+# build junit tests
+add_custom_target(
+  build-junit-tests
+  ALL
+  COMMAND
+    ${Java_JAVAC_EXECUTABLE} ${java_test_src_files}
+    -cp ${JUnit_JAR}:${CVC5_JAR_PATH} # add JUnit and cvc5 jar files to the class path
+    -d . # specify the output directory for the generated .class files
+  COMMENT "Build junit tests"
+  VERBATIM
+)
+
+# make sure junit jar file is present
+add_dependencies(build-junit-tests JUnit-EP-jar)
+# make sure cvc.jar file is built first
+add_dependencies(build-junit-tests cvc5jar)
+
```

Comment:
Here is the place where you should retrieve the information for `CVC5_JNI_PATH` and `CVC5_JAR_PATH` as described above.

Reply:
For some reason `get_target_property(CVC5_JAR_PATH cvc5jar JAR_FILE)` does not work and this error happens
`get_target_property() called with non-existent target "cvc5jar".`

Reply:
Are you able to call `get_target_property()` in `src/api/java/CMakeLists.txt`?

Reply:
Yes. It works fine there.

Reply:
Hmm that's weird since you obviously don't have a problem finding `cvc5jar` for `add_dependencies(build-junit-tests cvc5jar)`. Ok let's move the `get_target_property()` to the other file.

Reply:
Don.e My guess, add_jar target is local and not global if this makes sense.